src/sentry/static/sentry/app/components/events/interfaces/keyValueList/keyValueListV2.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import sortBy from 'lodash/sortBy';
+import styled from '@emotion/styled';
+
+import ContextData from 'app/components/contextData';
+import AnnotatedText from 'app/components/events/meta/annotatedText';
+import theme from 'app/utils/theme';
+
+type Props = {
+  data?: Array<KeyValueListData>;
+  onClick?: () => void;
+  raw?: boolean;
+  longKeys?: boolean;
+  isContextData?: boolean;
+  isSorted?: boolean;
+};
+
+export type KeyValueListData = {
+  key: string;
+  subject: React.ReactNode;
+  value: string | null;
+  meta: Meta;
+};
+
+type Meta = {
+  chunks: Array<Chunks>;
+  len: number;
+  rem: Array<Array<string | number>>;
+  err: Array<any>;
+};
+
+type Chunks = {
+  text: string;
+  type: string;
+  remark?: string;
+  rule_id?: string;
+};
+
+const KeyValueList = ({
+  data,
+  isContextData = false,
+  isSorted = true,
+  raw = false,
+  longKeys = false,
+  onClick,
+}: Props) => {
+  if (data === undefined || data === null || data.length === 0) {
+    return null;
+  }
+
+  const getData = () => {
+    if (isSorted) {
+      return sortBy(data, [({subject}) => subject]);
+    }
+    return data;
+  };
+
+  return (
+    <table className="table key-value" onClick={onClick}>
```

Comment:
Is removing the dependency on LESS styles something that you have planned for a future set of changes? It would be nice to further reduce our dependency on LESS in issue details.

Reply:
Yes.. it's in my plans :)